<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetEditorScreen</class>
 <widget class="QDialog" name="SetEditorScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="3" column="0">
    <widget class="QListWidget" name="setBrowser"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="highlightSetButton">
     <property name="text">
      <string>Highlight Selected Set</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Geom in Set</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="geomInSetBrowser"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Set Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="setNameInput"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sets</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="screenHeader">
     <property name="text">
      <string>Set Editor</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
